ui/src/app/kvm/components/KVMActionFormWrapper/ComposeForm/StorageTable/PoolSelect/PoolSelect.test.tsx

```diff
@@ -0,0 +1,225 @@
+import { act } from "react-dom/test-utils";
+import { mount } from "enzyme";
+import React from "react";
+import { MemoryRouter, Route } from "react-router-dom";
+import { Provider } from "react-redux";
+import configureStore, { MockStoreEnhanced } from "redux-mock-store";
+
+import type { Pod } from "app/store/pod/types";
+import type { RootState } from "app/store/root/types";
+import {
+  domainState as domainStateFactory,
+  fabricState as fabricStateFactory,
+  generalState as generalStateFactory,
+  podDetails as podDetailsFactory,
+  podState as podStateFactory,
+  podStatus as podStatusFactory,
+  podStoragePool as podStoragePoolFactory,
+  powerType as powerTypeFactory,
+  powerTypesState as powerTypesStateFactory,
+  resourcePoolState as resourcePoolStateFactory,
+  rootState as rootStateFactory,
+  spaceState as spaceStateFactory,
+  subnetState as subnetStateFactory,
+  vlanState as vlanStateFactory,
+  zoneState as zoneStateFactory,
+} from "testing/factories";
+import ComposeForm from "../../ComposeForm";
+
+const mockStore = configureStore();
+
+const generateWrapper = (store: MockStoreEnhanced, pod: Pod) =>
```

Comment:
Do we need `MockStoreEnhanced` rather than `MockStore`?

Reply:
Seems to be fine to use `MockStore`